README.md

```diff
@@ -1,15 +1,21 @@
 # img-site
-This repo hosts the files for my images/quotes webpage!
 
-To add quotes/images: create a PR and add the image!
+This repo hosts the files for my images/quotes webpage! https://img.kaashout.tech
```

Comment:
new domain is kaashout.me

Reply:
> new domain is kaashout.me

GH desktop told me that it saved the new domain when I changed it ig those changes didnt reflect